### PR TITLE
Add Pipfile and Pipfile.lock

### PR DESCRIPTION
**Pull request by dhavalsavalia**
I have closed previous PR because I messed up commits and was not able to resolve the issue. I am still learning things. So I have create raise this new PR.

### CODE REVIEW

**Comment by dhavalsavalia** [comment]
Updated README.md with instructions to work with pipenv

**Comment by userimack** [comment]
I would like to work on the issue.

**Comment by dhavalsavalia** [comment]
Any updates here?

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pelican = "==3.5.0"
+markdown = "*"
```
Comment (realslimshanky) [inline Pipfile]: Please specify a version.

Comment (dhavalsavalia) [reply Pipfile]: There is no `markdown` version specified in `requirements.txt`, so I am going with the latest version that is working without any issues, i.e. `2.6.11`

**Comment by ananyo2012** [comment]
Someone needs to verify the steps @pythonindia/pycon-2018-team

**Comment by realslimshanky** [comment]
@ananyo2012 steps to install pipenv and dependencies using pipenv looks good. All what's missing is specific version in the Pipfile for markdown.

**Comment by realslimshanky** [comment]
Thank you for your contribution @dhavalsavalia :taco:

**Pull request merged**